packages/common/src/worker/devToolWorker.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+let devToolWorker: Worker | null = null;
+
+/**
+ * Get the devTool worker instance. If it doesn't exist, it creates a new one.
+ * @returns {Worker} The DevTool worker instance.
+ */
+const getDevToolWorker = (): Worker => {
```

Comment:
Correct me if I am wrong but this utility will only work if there is a worker.js file among the build artifacts, which is just the case for build artifacts of the devtools panel. 
That makes the dir `packages/extension/src/utils/` a better place for this utility since it cannot be used anywhere else.

Reply:
Since extension package needs to import items from library-detection package, we cannot move it to the extension package as it will create circular dependency ( as library-detection package will have to import from extension too )